From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Victor Gomes <victorgomes@chromium.org>
Date: Fri, 31 Jul 2026 16:05:08 +0200
Subject: [M150] [turbofan] Disable additive safe integer feedback

Original change's description:
> [turbofan] Disable additive safe integer feedback
>
> Bug: 539350801
>
> Change-Id: I679b5120df2d736da823f7420b2432df31302d6c
> Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/8174345
> Auto-Submit: Victor Gomes <victorgomes@chromium.org>
> Commit-Queue: Victor Gomes <victorgomes@chromium.org>
> Reviewed-by: Leszek Swirski <leszeks@chromium.org>
> Commit-Queue: Leszek Swirski <leszeks@chromium.org>
> Cr-Commit-Position: refs/heads/main@{#108981}

(cherry picked from commit ab2cad06c8e209248ce0d2cf76e1f0d16aa51533)

Bug: 546215601,539350801
Change-Id: I679b5120df2d736da823f7420b2432df31302d6c
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/8253364
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Commit-Queue: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Auto-Submit: chrome-cherry-picker@chops-service-accounts.iam.gserviceaccount.com <chrome-cherry-picker@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/15.0@{#61}
Cr-Branched-From: 0c6c73eea6994260f0a1d88469708360ea60c660-refs/heads/15.0.245@{#1}
Cr-Branched-From: 03d64f40934c058cd1c6d7e5cead5b757a4faedd-refs/heads/main@{#107679}

diff --git a/src/flags/flag-definitions.h b/src/flags/flag-definitions.h
index 5d535728add8806add8e0d062c8d19b5443e241b..b99fff1b2a80ae6687b69199553b22bccc65e202 100644
--- a/src/flags/flag-definitions.h
+++ b/src/flags/flag-definitions.h
@@ -858,7 +858,7 @@ DEFINE_EXPERIMENTAL_FEATURE(
     "strings.")
 
 #ifdef V8_TARGET_ARCH_64_BIT
-DEFINE_BOOL(additive_safe_int_feedback, true,
+DEFINE_BOOL(additive_safe_int_feedback, false,
             "Enable the use of AdditiveSafeInteger feedback")
 DEFINE_EXPERIMENTAL_FEATURE(
     turbolev_additive_safe_int_feedback,
